modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -0,0 +1,151 @@
+
+NOTE: The JSON Schema format is not supported for Iceberg topics. If your topic data is in JSON, use the `key_value` mode. 
+
+== Supported Iceberg modes
+
+Redpanda supports the following modes for Iceberg topics:
+
+=== key_value
+
+Creates an Iceberg table using a simple schema, consisting of two columns, one for the record metadata including the key, and another binary column for the record's value.
+
+=== value_schema_id_prefix
+
+Creates an Iceberg table whose structure matches the Redpanda schema for the topic, with columns corresponding to each field. You must register a schema in the xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] and producers must write to the topic using the Schema Registry wire format.
+
+ifndef::env-cloud[]
+In the Schema Registry wire format, a "magic byte" and schema ID are embedded in the message payload header. Producers to the topic must use the wire format in the serialization process so Redpanda can determine the schema used for each record, use the schema to define the Iceberg table, and store the topic values in the corresponding table columns. See also: xref:{schema-id-val-doc}[] and 
+the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post.
+
+ifdef::env-cloud[]
+In the Schema Registry wire format, a "magic byte" and schema ID are embedded in the message payload header. Producers to the topic must use the wire format in the serialization process so Redpanda can determine the schema used for each record, use the schema to define the Iceberg table, and store the topic values in the corresponding table columns. See also: the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post.
+endif::[]
+
+=== value_schema_latest
+
+Creates an Iceberg table whose structure matches the latest schema registered for the subject in the Schema Registry. You must register a schema in the xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry]. Unlike the `value_schema_id_prefix` mode,  `value_schema_latest` does not require that producers use the wire format.
+
+=== disabled
+
+Default for `redpanda.iceberg.mode`. Disables writing to an Iceberg table for the topic.
+
+== Configure Iceberg mode for topic
+
+You can set the Iceberg mode for a topic when you create the topic, or you can update the mode for an existing topic.
+
+.Create a new topic and set `redpanda.iceberg.mode`:
```

Comment:
This renders in italics--not sure it is intended?

Reply:
Added "Option 1" and "Option 2" to better indicate block title